MIP/MIP-60/README.md

```diff
@@ -0,0 +1,234 @@
+# MIP-60: Crosschain bridge architectures
```

Comment:
Since this is mainly definitional, I would suggest framing this as an MIP which introduces MGs.